### PR TITLE
WIP : fix CTF lcmv due to comps

### PR DESCRIPTION
**Pull request by agramfort**
See: https://gist.github.com/Heneark/a183a24218302831860efc86c84c1950
and https://github.com/mne-tools/mne-python/issues/5292#issuecomment-404176118

cc @Heneark

### CODE REVIEW

File: mne/beamformer/_compute_beamformer.py [hunk 1]
```diff
@@ -116,7 +116,10 @@ def _compare_ch_names(names1, names2, bads):
 
 def _check_one_ch_type(info, picks, noise_cov):
     """Check number of sensor types and presence of noise covariance matrix."""
+    comps = info['comps']
+    info['comps'] = []
     info_pick = pick_info(info, sel=picks)
+    info['comps'] = comps
```
Comment (agramfort) [inline mne/beamformer/_compute_beamformer.py]: @larsoner do you have a nicer suggestion?

Comment (larsoner) [reply mne/beamformer/_compute_beamformer.py]: For now maybe just put a `# XXX this is a hack that should be refactored` until we converge and tackle #5342

**Comment by agramfort** [comment]
@Heneark can you test current master?

**Comment by agramfort** [comment]
backported

**Pull request merged**